cpp/src/arrow/compute/exec/key_hash.cc

```diff
@@ -138,6 +138,19 @@ inline uint32_t Hashing::helper_tail(uint32_t offset, uint64_t mask, const uint8
   return acc;
```

Comment:
Same here: You can make this method call the other one specifying length as `sizeof(uint64_t)`

Reply:
Fixed